components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -80,8 +81,28 @@ export class CreateWorkspace extends React.Component<CreateWorkspaceProps, Creat
     }
 
     else if (result?.existingWorkspaces) {
-      // FIXME Force create
-      statusMessage = <div className="text-base text-gray-400">Existing workspaces:<ul>{result.existingWorkspaces.map(w => <li>→ <a className="text-blue" href={w.latestInstance?.ideUrl}>{w.workspace.id}</a></li>)}</ul></div>;
+      statusMessage = <Modal visible={true} closeable={false}>
+        <h3>Running Workspaces</h3>
+        <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-2">
+          <p className="mt-1 mb-2 text-base">You already have running workspaces with the same context. You can open an existing one or open a new workspace.</p>
+          <>
+            {result?.existingWorkspaces?.map(w =>
+              <a href={w.latestInstance?.ideUrl} className="rounded-xl hover:bg-gray-100 flex p-3 my-1">
+                <div className="w-full">
+                  <p className="text-base text-black font-bold">{w.workspace.id}</p>
+                  <p>{w.workspace.contextURL}</p>
+                </div>
+                <div className="flex">
+                  <button className="px-3 py-1 my-auto bg-green hover:bg-green-dark border-green-darker">Open</button>
```

Comment:
**issue**: Didn't notice this earlier because there was only one workspace in this list but the _Open_ button should become visible only when hovering over each list item. 😅

Reply:
~~I actually knew that but got lazy~~ Nice catch!